docs/guide/resolving_proxy_contracts.rst

```diff
@@ -37,6 +37,10 @@ By default, ``proxy_config`` uses a configuration with two **ProxyChecks**:
  - ArgentProxyCheck - resolves `Argent Proxy <https://github.com/argentlabs/argent-contracts-starknet/blob/b7c4af7462a461386d29551400b985832ba942de/contracts/upgrade/Proxy.cairo>`_.
  - OpenZeppelinProxyCheck - resolves `OpenZeppelin Proxy <https://github.com/OpenZeppelin/cairo-contracts/blob/d12abf335f5c778fd19d6f99e91c099b40865deb/src/openzeppelin/upgrades/presets/Proxy.cairo>`_.
 
+.. warning::
+
+    ``StarknetEthProxyCheck`` has been removed, because StarkGate ETH Token contract changes the type of interaction. Currently, StarkGate * Token contracts should be used as ERC20.
```

Comment:
What do you mean exactly by `changes the type of interaction`? I believe we should simplify this information, emphasizing that the ETH contract has been updated (to Cairo 2) and as it's no longer a Proxy, there's no need to use `proxy_config` in the `Contract` class.

Reply:
`type of interaction` was taken from Starkscan, but they changed the description of `Type` row! Your proposition is better